### ISSUE TITLE
Nested double quotes break highlighting in .eex files

### ISSUE THREAD

**Issue by optikfluffel** [elixir-editors/elixir-tmbundle#183]
Example from a newly setup Phoenix project with broken highlighting on Lines 8, 20 and 29:
![Screenshot 2020-03-08 at 11 07 50](https://user-images.githubusercontent.com/379915/76160857-ff210c80-612d-11ea-9494-bbeab70d0ddb.png)

**Comment by josevalim** [comment]
Yeah, this has been annoying the hell out of me. :/ but I haven't had the time to take a proper look at it yet. If you could take a look at it, it would be great! :heart:
